src/Process/PointCloudProcessing.js

```diff
@@ -163,11 +166,7 @@ export default {
         // only load geometry if this elements has points
         if (elt.numPoints > 0) {
             if (elt.obj) {
-                elt.obj.material.visible = true;
-                elt.obj.material.size = layer.pointSize;
-                if (elt.obj.material.updateUniforms) {
-                    elt.obj.material.updateUniforms();
-                }
+                elt.obj.material.copy(layer.material);
```

Comment:
this implies `elt.obj.material.visible = true;` if the `layer.material` is visible